src/quickgui/plugin/qgsquickmapcanvas.qml

```diff
@@ -0,0 +1,142 @@
+
+/***************************************************************************
+ qgsquickmapcanvas.qml
+  --------------------------------------
+  Date                 : 10.12.2014
+  Copyright            : (C) 2014 by Matthias Kuhn
+  Email                : matthias@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+import QtQuick 2.3
+import QtQuick.Controls 2.2
+import QtQml 2.2
+import QgisQuick 0.1 as QgsQuick
+
+Item {
+  id: mapArea
+  property alias mapSettings: mapCanvasWrapper.mapSettings
+  property alias isRendering: mapCanvasWrapper.isRendering
```

Comment:
Would be great to add some (minimal) docs here, including that `isRendering` is read only while the other two are read/write. And maybe about the effect of `incrementalRendering` w.r.t. labeling.